crates/orderbook/src/orderbook.rs

```diff
@@ -147,11 +164,13 @@ impl From<ValidationError> for ReplaceOrderError {
     }
 }
 
+/*
 impl From<InsertionError> for ReplaceOrderError {
     fn from(err: InsertionError) -> Self {
         Self::Add(err.into())
     }
 }
+*/
```

Comment:
Why leave commented code? Why not just remove it? Are you planning to do something with it later?

Reply:
Should be removed. Accidentally left in.